src/template.html

```diff
@@ -20,124 +20,122 @@
 
   <!-- Hero -->
   <div id="hero">
-    <p id="countdown">This free web app will expire in
-      <strong id="timeLeft"></strong>
+    <p id="countdown">This <strong>free</strong> web application will expire in
+      <strong id="timeLeft">...</strong>
+      minutes...
     </p>
     <img src="./images/node-logo.svg" alt="Azure App Service">
-    <h1>🎉 Welcome to Express on Azure App Service 🎉</h1>
-    <h2>Hosted at
-      <a href="/" id="host"></a>
-    </h2>
+    <h1> Welcome to Azure! </h1>
+    <p>This is a simple Node application using the popular Express framework, hosted on Azure App Service at <a href="/" id='host'></a>.</p>
+    <p>To get started, follow the instructions below or jump straight to the <a href="https://code.visualstudio.com/tutorials/app-service-extension/getting-started" target="_blank">walkthrough</a>.</p>
   </div>
 
-  
+
   <!-- Content -->
   <div>
-    <div class="content-wrapper-light">
-      <div class="content">
-        <div class="column">
-          <div>
-            <p class="title">Get started</p>
-              <p> 1. Clone the code locally using one of the options to the right.</p>
-          </div>
-        </div>
-      
-        <div class="column vsc">
-          <div class="icon">
-            <img src="./images/vsc-logo.svg" alt="VS Code">
-          </div>
-          <div class="text">
-            <p class="title">Start editing with VS Code</p>
-            <p class="description">Easily edit and deploy straight from our favorite editor.</p>
-            <p class="link">
-              <a href="#"
-                id="clone">Clone with VS Code</a>
-            </p>
-            <p class="link">
-              <a href="#"
-                id="clone-insiders">Clone with VS Code Insiders</a>
-            </p>
-          </div>
-        </div>
-      
-        <div class="column git">
-          <div class="icon">
-            <img src="./images/git-logo.svg" alt="Git">
-          </div>
-          <div class="text">
-            <p class="title">Clone or push with Git</p>
-            <p class="description">Use git command line and your favorite editor to make changes and deploy to Azure.</p>
-            <p class="link">
-              <a href="#" id="show-creds">Generate your Git credentials</a>
-            </p>
-            <p id="creds" style="display: none"></p>
-      
-          </div>
-        </div>
-      </div>
-    </div>
+
 
     <div class="content-wrapper">
       <div class="content">
         <div class="column">
-          <div>
-              <p>2. Make a text change in <b>template.html</b></p>
+          <div class="content">
```

Comment:
This is the same class being used two levels above, so you're inheriting a lot of duplicate styles.